libraries/AC_AutoTune/AC_AutoTune_Heli.cpp

```diff
@@ -706,6 +706,31 @@ void AC_AutoTune_Heli::save_tuning_gains()
     reset();
 }
 
+// send final gains for a given axis to GCS
+void AC_AutoTune_Heli::send_final_gains(AxisType test_axis) const
+{
+    switch (test_axis) {
+        case ROLL:
+            send_axis_gains("Roll", tune_roll_rp, tune_roll_rff*AUTOTUNE_FFI_RATIO_FINAL, tune_roll_rd, tune_roll_rff, tune_roll_sp, tune_roll_accel);
+            break;
+        case PITCH:
+            send_axis_gains("Pitch", tune_pitch_rp, tune_pitch_rff*AUTOTUNE_FFI_RATIO_FINAL, tune_pitch_rd, tune_pitch_rff, tune_pitch_sp, tune_pitch_accel);
+            break;
+        case YAW:
+            send_axis_gains("Yaw", tune_yaw_rp, tune_yaw_rp*AUTOTUNE_YAW_PI_RATIO_FINAL, tune_yaw_rd, tune_yaw_rff, tune_yaw_sp, tune_yaw_accel);
+            break;
+    }
+}
+
+// send gain formating helper
+void AC_AutoTune_Heli::send_axis_gains(const char* axis_string, float rate_P, float rate_I, float rate_D, float rate_ff, float angle_P, float max_rate) const
+{
+    gcs().send_text(MAV_SEVERITY_NOTICE,"AutoTune: %s complete", axis_string);
+    gcs().send_text(MAV_SEVERITY_NOTICE,"AutoTune: %s Rate: P: %0.2f, I: %0.2f, D: %0.4f, FF: %0.2f",axis_string,rate_P,rate_I,rate_D,rate_ff);
```

Comment:
two digits is enough for P and I?  I would have chosen 3 and removed the space  between "P:" and the number so it doesn't get any longer.

Reply:
Gone to 3 digits for P I and FF